src/tile/routes/tileRouter.ts

```diff
@@ -0,0 +1,13 @@
+import { container } from 'tsyringe';
+import { FastifyPluginRegister } from '../../common/interfaces';
+import { TileController } from '../controllers/tileController';
+
+const tileRoutesRegistry: FastifyPluginRegister = (fastify, _, done) => {
+  const controller = container.resolve(TileController);
+
+  fastify.get('/:z(\\d+)/:x(\\d+)/:y(\\d+).png', controller.getTile);
```

Comment:
to catch minus sign:
```suggestion
  fastify.get('/:z(-?\\d+)/:x(-?\\d+)/:y(-?\\d+).png', controller.getTile);
```

Reply:
changed to `(^\\d+)`